ReadMe.md

```diff
@@ -95,4 +96,4 @@ to talk to the developers in the `#gdext-dev` channel on [Discord]!
 [Godot]: https://godotengine.org
 [`gdnative`]: https://github.com/godot-rust/godot-rust
 [mpl]: https://www.mozilla.org/en-US/MPL/
-[Discord]: https://discord.gg/aKUCJ8rJsc
\ No newline at end of file
+[Discord]: https://discord.gg/aKUCJ8rJsc
```

Comment:
What changed here? 🙂

Reply:
The addition of a new line, I think. I edited it within github so it's probably the github editor formatting the text.